executor/version.go

```diff
@@ -0,0 +1,5 @@
+package executor
+
+var (
+	TitusExecutorVersion = ""
```

Comment:
How about `ManualBuild` so that if it is unset we have that sentinel value? (we know that it didn't go through CI)